service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/controller/utils/CustomXMLException.java

```diff
@@ -0,0 +1,13 @@
+package uk.nhs.adaptors.oneoneone.cda.report.controller.utils;
```

Comment:
utils is not the best package for this class
Maybe let's create a dedicated exceptions package?